assets/src/scss/pages/_listing-page.scss

```diff
@@ -0,0 +1,153 @@
+@mixin render-bullet {
+  &:before {
+    display: block;
+    margin: 0 $sp-1;
+    content: "•";
+  }
+}
+
+.query-list-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-top: var(--query-list--item--margin-top, $sp-4);
+
+  @include medium-and-up {
+    flex-direction: row;
+  }
+}
+
+.query-list-item-image {
+  flex-basis: 30%;
+
+  &.query-list-item-image-max-width {
+    img {
+      width: 100%;
+      height: 310px;
+      object-fit: cover;
+      object-position: 50% 50%;
+      margin-right: $sp-2;
+
+      html[dir="rtl"] & {
+        margin-right: 0;
+        margin-left: $sp-2;
+      }
+
+      @include medium-and-up {
+        height: 200px;
+      }
+
+      @include large-and-up {
+        height: 250px;
+      }
+
+      @include x-large-and-up {
+        height: 260px;
+      }
+
+      @include xx-large-and-up {
+        height: 240px;
+      }
+    }
+  }
+
+  .wp-post-image {
+    height: 100%;
+  }
+}
+
+.query-list-item-body {
+  flex-basis: 70%;
+  flex-shrink: 0;
+  margin-top: $sp-1x;
+
+  header {
+    margin-top: $sp-1;
+  }
+
+  // Chrome and Safari
```

Comment:
It's not 100% supported by Firefox, but from version `103` (currently `106.*`) it's been added as an experimental feature which can be enabled if go to the Firefox `about:config` page, then search and toggle `layout.css.has-selector.enabled`.

More info:
https://developer.mozilla.org/en-US/docs/Web/CSS/:has